packages/sdks/e2e/tests/src/tests/main.spec.ts

```diff
@@ -57,4 +70,54 @@ test.describe(targetContext.name, () => {
       text: 'Mattel Certified by Great Place to Work and Named to Fast Company’s List of 100 Best Workplaces for Innovators',
     });
   });
+
+  test.describe('custom-breakpoints', () => {
```

Comment:
would it be possible to add one set of tests for where the custom breakpoints are not set, so we test the default flow as well and ensure nothing is broken there?

Reply:
done